### PR TITLE
Frontend: Add Service (as Admin) + View Services (everyone)

### LINKED ISSUE #169: Service tab: Add Service (Admin)
### **🏁 Setup**
- [x] **Assign** yourself to this issue 

### **✅ To do:**
- [x] **Draw** page mockup using [Whimsical](https://whimsical.com/repairshop-6v7GHoRVFfSSvyvUdeHaQ8) 
(After clicking on the link, you must sign in with your Google Account)

**Frontend**
- [x] Text fields for `name`, `duration`, `price`
- [x] Implement frontend

### **🆘 If you need help for the issue you’re working on:**

**1) Write** comments in this issue 
**2) Add** label image
**3) Copy** the issue link and **paste** it to Discord's `#🐛issues` channel
**4)** When solved, **strike** the message in `#🐛issues`, **mark it** as `(SOLVED)`  + **remove** label `help wanted` + **write** comments in this issue 

### *Whenever something is done write a comment below.*

### LINKED ISSUE #170: Service tab: View Services (everyone)
### **🏁 Setup**
- [x] **Assign** yourself to this issue 

### **✅ To do:**
- [x] **Draw** page mockup using [Whimsical](https://whimsical.com/repairshop-6v7GHoRVFfSSvyvUdeHaQ8) 
(After clicking on the link, you must sign in with your Google Account)

**Frontend**

- [x] Display all services 
- [x] If no services, display no service
- [x] Implement frontend

### **🆘 If you need help for the issue you’re working on:**

**1) Write** comments in this issue 
**2) Add** label image
**3) Copy** the issue link and **paste** it to Discord's `#🐛issues` channel
**4)** When solved, **strike** the message in `#🐛issues`, **mark it** as `(SOLVED)`  + **remove** label `help wanted` + **write** comments in this issue 

### *Whenever something is done write a comment below.*

### PR DESCRIPTION
**Pull request by sandyl289**
Branch: `frontend/add-service`

- All tests pass (Repository & Service) _I ran the tests just in case_
- `Add Service` (as Admin) (https://github.com/McGill-ECSE321-Winter2021/project-group-09/issues/169)
- `View Services` (everyone) (https://github.com/McGill-ECSE321-Winter2021/project-group-09/issues/170)
- Change service param to ServiceDto
- Modified navbar to only let Admin see "Add Service"

Thanks Tyler for helping me! 

# Add New Service (Must be logged in as Admin)

## Add New Service
image

## Successful Message
image

## If service name already exists
image

# View Services

## If no service yet
image

## If there are services
image

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (T-Syme) [inline .vscode/launch.json:15]: Interesting that this file isn't included in the .gitignore... It shouldn't matter though

**Review by T-Syme — APPROVED** [review]

File: RepairShop-Frontend/src/components/AddService.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+var config = require("../../config");
+var AXIOS = axios.create({
+  //DEVELOPMENT
+  baseURL: "http://" + config.dev.backendHost + ":" + config.dev.backendPort,
+
+  //PRODUCTION
+  //baseURL: "http://" + config.build.backendHost + ":" + config.build.backendPort,
+
+  headers: {
```
Comment (JDziewonski98) [inline RepairShop-Frontend/src/components/AddService.vue]: Does the call work without this?

File: RepairShop-Frontend/src/components/Logout.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="titleContainer">
     <h1 v-show="success">Logged Out Successfully!</h1>
-    <h1 v-show="!success">Your were not logged in.</h1>
+    <h1 v-show="!success">You were not logged in.</h1>
```
Comment (JDziewonski98) [inline RepairShop-Frontend/src/components/Logout.vue:4]: hmm why is this stuff showing up? I though I already changes this on main

File: RepairShop-Frontend/src/components/AddService.vue [hunk 4]
```diff
@@ -0,0 +1,133 @@
+
+<template>
+  <div id="addServiceForm">
+    <h2>Add New Service</h2>
+
+    <p>
+      <span v-if="successAddService" style="color: green">
+        {{ successAddService }}
```
Comment (JDziewonski98) [inline RepairShop-Frontend/src/components/AddService.vue]: I find this a bit hard to notice, maybe make it a bolder color or make it flash? Not a big deal at all

**Review by JDziewonski98 — APPROVED** [review]
Looks good to me, nice work

**Comment by sandyl289** [comment]
I'll close https://github.com/McGill-ECSE321-Winter2021/project-group-09/issues/169 and https://github.com/McGill-ECSE321-Winter2021/project-group-09/issues/170

**Pull request merged**